test/optimizer/rules/test_rules.py

```diff
@@ -207,15 +225,16 @@ def test_supported_rules(self):
             LogicalJoinToPhysicalHashJoin(),
             LogicalCreateMaterializedViewToPhysical(),
             LogicalFilterToPhysical(),
-            LogicalProjectToPhysical(),
+            DistributedLogicalProjectToPhysical()
+            if ray_enabled
+            else SequentialLogicalProjectToPhysical(),
```

Comment:
Could we add a comment that indicates right now `DistributedLogicalProjectToPhysical()` and `SequentialLogicalProjectToPhysical()` are the same?